src/app/static/src/app/store/adr/actions.ts

```diff
@@ -7,24 +7,25 @@ import {ADRMutation} from "./mutations";
 import {datasetFromMetadata} from "../../utils";
 import {Organization, Release} from "../../types";
 import {BaselineMutation} from "../baseline/mutations";
+import {DataExplorationState} from "../dataExploration/dataExploration";
 
 export interface ADRActions {
-    fetchKey: (store: ActionContext<ADRState, RootState>) => void;
-    saveKey: (store: ActionContext<ADRState, RootState>, key: string) => void;
-    deleteKey: (store: ActionContext<ADRState, RootState>) => void;
-    getDatasets: (store: ActionContext<ADRState, RootState>) => void;
-    getReleases: (store: ActionContext<ADRState, RootState>, id: string) => void;
-    getDataset: (store: ActionContext<ADRState, RootState>, payload: GetDatasetPayload) => void;
-    getSchemas: (store: ActionContext<ADRState, RootState>) => void;
-    getUserCanUpload: (store: ActionContext<ADRState, RootState>) => void;
+    fetchKey: (store: ActionContext<ADRState, DataExplorationState>) => void;
```

Comment:
is there a reason why you change RootState to DataExplorationState considering RootState already extends DataExplorationState ?

Reply:
Yes, these actions will be invoked by both apps, so sometimes the state will be of the type `DataExplorationState`, and sometimes it will be the superset `RootState` (but will only invoke properties on the subset included in `DataExplorationState`)